zap/src/output/luau/client.rs

```diff
@@ -563,68 +564,72 @@ impl<'src> ClientOutput<'src> {
 			self.push_stmts(statements);
 		}
 
-		if ev.call == EvCall::SingleSync || ev.call == EvCall::SingleAsync {
-			self.push_line(&format!("if unreliable_events[{id}] then"));
+		if ev.call == EvCall::Polling {
+			self.push_polling_event(ev)
 		} else {
```

Comment:
Why is this different to how reliable polling events are pushed?
Shouldn't unreliable/reliable events should have (roughly) the same codepath for listeners?